modules/sre-cluster-access.adoc

```diff
@@ -64,7 +64,9 @@ Each of these access types have different levels of access to components:
 == SRE access to AWS accounts
 Red Hat personnel do not access AWS accounts in the course of routine {product-title} operations. For emergency troubleshooting purposes, the SREs have well-defined and auditable procedures to access cloud infrastructure accounts.
 
-SREs generate a short-lived AWS access token for a reserved role using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. Both STS and non-STS clusters use the AWS STS service for SRE access. For non-STS clusters, the `BYOCAdminAccess` role has the `AdministratorAccess` IAM policy attached, and this role is used for administration. For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached, and this role is used for administration.
+In the isolated backplane flow, SREs request access to a customer's support role. This request is just-in-time (JIT) processed by the backplane API which dynamically updates the organization role's permissions to a specific SRE personnel's account. This SRE's account is given access to a specific Red Hat customer's environment. SRE access to a Red Hat customer's environment is a temporary, short-lived access that is only established at the time of the access request. 
+
+Access to the STS token is audit-logged and traceable back to individual users. Both STS and non-STS clusters use the AWS STS service for SRE access. Access control uses the unified backplane flow when the `ManagedOpenShift-Technical-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached, and this role is used for administration. Access control uses the isolated backplane flow when the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Technical-Support-<org_id>` policy attached. See the KCS article link:https://access.redhat.com/solutions/7045629[Updating Trust Policies for ROSA clusters] for more information.
```

Comment:
Should this type of access be related back to a specific access type under the "Privileged access controls in ROSA" section starting on line 40, e.g. AWS access or elevation? Or is that a different type of access again?

It might just be me, but the flow here isn't helping me to connect the various ideas. Wondering if something like this might work:

> When emergency troubleshooting is required, authenticated SRE personnel can request access to a specific support role in the customer organization. When this request is made, the backplane API adds the customer organization and a specific support role and access policy to that SRE personnel's account. This allows that SRE personnel temporary and short-lived access to that role in that environment using the AWS Secure Token Service (STS). Access via STS token is audit-logged and traceable back to individual users.

The "isolated backplane" vs "unified backplane" aren't defined as far as I can see, either - we could probably spell out what those are alongside which role and policy they are used for, before linking off to the KCS article.